### PR TITLE
feat(.travis.yml): Creating Travis-CI config

### PR DESCRIPTION
**Pull request by GMishx**
Currently the script installs Atarashi and run each agent once as we do not have test scripts in place.

Need to be merged after #48

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright Siemens AG, 2019
+# SPDX-License-Identifier: GPL-2.0
```
Comment (maxhbr) [inline .travis.yml]:
```suggestion
# SPDX-License-Identifier: GPL-2.0-only
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+before_install:
+  - pip install --upgrade pip
+  - pip -V
+
+install:
+  - pip install .
+
+script:
+  - atarashi -h
+  - atarashi -a Ngram -s BigramCosineSim ./atarashi/atarashii.py
```
Comment (maxhbr) [inline .travis.yml]: off-topic: `atarashi` vs `atarashii`. What is the correct way to write it?

Comment (maxhbr) [reply .travis.yml]: These tests do not contain any assertions like a `grep` fro the expected result.

Comment (GMishx) [reply .travis.yml]: > off-topic: atarashi vs atarashii. What is the correct way to write it?

`atarashii.py` is the actual entry point and `atarashi` will be the symlink in `$PATH` location. It was named so to remove confusion between module `atarashi` and file `atarashi.py`.

> These tests do not contain any assertions like a grep fro the expected result.

I did not think of doing that here as atarashi is still not stable enough for functional testing.
I am just running all the agents to make sure new changes does not break flow of atarashi.

File: .travis.yml (hunk 1 above)
Comment (GMishx) [reply .travis.yml]: Done.

**Pull request merged**